tests/src/vm_host_env/ibc.rs

```diff
@@ -268,26 +260,11 @@ pub fn init_storage() -> (Address, Address) {
     (token, account)
 }
 
-pub fn tm_dummy_header() -> TmHeader {
-    TmHeader {
-        version: TmVersion { block: 10, app: 0 },
-        chain_id: TmChainId::try_from("test_chain".to_owned())
-            .expect("Creating an TmChainId shouldn't fail"),
-        height: TmHeight::try_from(10_u64)
-            .expect("Creating a height shouldn't fail"),
-        time: TmTime::now(),
-        last_block_id: None,
-        last_commit_hash: None,
-        data_hash: None,
-        validators_hash: TmHash::None,
-        next_validators_hash: TmHash::None,
-        consensus_hash: TmHash::None,
-        app_hash: AppHash::try_from(vec![0])
-            .expect("Creating an AppHash shouldn't fail"),
-        last_results_hash: None,
-        evidence_hash: None,
-        proposer_address: TmAccountId::try_from(vec![0u8; 20])
-            .expect("Creating an AccountId shouldn't fail"),
+pub fn tm_dummy_header() -> anoma::types::storage::Header {
+    anoma::types::storage::Header {
+        hash: anoma::types::hash::Hash([0; 32]),
+        time: TmTime::now().try_into().unwrap(),
```

Comment:
not directly related to this PR I think we can update `TryFrom<DateTimeUtc> for Time` and `TryFrom<Time> for DateTimeUtc` to just be `From` traits, it doesn't look like they should ever error, so this `.unwrap()` is OK